src/annotations/components/controlBar/AnnotationPill.tsx

```diff
@@ -21,6 +22,10 @@ export const AnnotationPill: FC<Props> = ({id, name, description, color}) => {
   const handleDisable = (): void => {
     dispatch(disableAnnotationStream(id))
   }
+  const handleDeleteClick = (): void => {
+    console.log('hi im delete')
+    deleteAnnotations(id)
```

Comment:
so since I suggested we remove the `catch` block from the thunk above, let's add it in here:

```ts
try {
  deleteAnnotations(id)
  dispatch(notify(deleteAnnotationSuccess()))
} catch (error) {
  dispatch(notify(deleteAnnotationFailed(error.message)))  
}
```

the reason I think this makes more sense is because this creates a separation of concerns between redux and the display layer. If there's a problem in redux, it'll just throw the error and let the view layer that's using it handle it. The view layer can then get the error and handle it in a nice and helpful way.